addons/binding/org.openhab.binding.tankerkoenig/README.md

```diff
@@ -110,16 +110,56 @@ java.io.IOException: java.util.concurrent.ExecutionException: javax.net.ssl.SSLH
 ......
 ```
 
-That indicates a missing certificate for the https-connection on the system.
-In order to get the required certificate on a Linux-system one needs to perform these steps:
+That indicates a missing certificate of a certification authority (CA) in the certificate-store of the java jdk under which openHAB is running.
+In most cases an update to the latest jdk version does solve this, because the store of the cacerts are maintained with the java versions.
+In other words, while doing an update of the java version the missing certificate will be installed.
+After the java-update openhab2 has to be restarted.
 
+When running an old linux installation which has only old java-versions in the package-repositories, there are three possibilities:
+
+   1.) Update the linux-system and install the latest java-versions
+   
+   2.) Install a newer java-version on a different way
```

Comment:
Consider, "2.) Download the most recent jdk and install it directly on to your system without using a pre-composed package